### DISCUSSION TITLE
how to relate two Selects?

### DISCUSSION THREAD

**Discussion by FrancoPaez** [JedWatson/react-select#4772] [Q&A]
Could you give me a hand with this? I have to enable a Select react only if the previous one was already used, but I don't really understand the methods as the documentation shows them

![image](https://user-images.githubusercontent.com/82821596/133118693-81fe8b8e-951a-4904-a960-707958c67f62.png)

Thank you

**Comment by hermbo** [comment]
@FrancoPaez Your going to want to use state to store your selections so that you can check state to see if anything is selected.

Example:
```javascript
const [state, setState] = useState({
    areaSelect: [],
    subAreaSelect: []
  })

const handleSelect = (value, action) => {
    setState( ...state, [action.name]: value })
  }

return (

<Select
   name={'areaSelect'}
   value={state.areaSelect}
   options={yourOptions}
   onChange={(value, action) => handleSelect(value, action)}
 />

<Select
   name={'subAreaSelect'}
   value={state.subAreaSelect}
   options={yourOptions}
   onChange={(value, action) => handleSelect(value, action)}
   isDisabled={!state.areaSelect.length}
 />
) 
```

You'll see that I am checking the state array `isDisabled={!state.areaSelect.length}` to see if there is a length, if yes then it's enabled, if not then it's disabled.
